ddl/syncer.go

```
@@ -175,6 +180,32 @@ func (s *schemaVersionSyncer) GlobalVersionCh() clientv3.WatchChan {
 	return s.globalVerCh
 }
 
+// WatchGlobalSchemaVer implements SchemaSyncer.WatchGlobalSchemaVer interface.
+func (s *schemaVersionSyncer) WatchGlobalSchemaVer(ctx context.Context, tryCnt int) error {
+	for i := 0; i < tryCnt; i++ {
+		ch := s.etcdCli.Watch(ctx, DDLGlobalSchemaVersion)
+		var err error
+		select {
+		case resp, ok := <-ch:
+			if !ok || resp.Err() != nil {
+				err = resp.Err()
+				break
+			}
+			// This situation should not happen.
+			s.globalVerCh = ch
+			log.Warnf("[syncer] watch global schema finished, but this situation should not happen")
```

If this is an unexpected situation, why not return an error?